.travis.yml

```diff
@@ -1,5 +1,63 @@
-language: node_js
-node_js:
-  - "node"
+sudo: false
+
+language:
+  - php
+  - node_js
+
+notifications:
+  email:
+    on_success: never
+    on_failure: change
+
+branches:
+  only:
+    - master
+
+cache:
+  directories:
+    - vendor
+    - $HOME/.composer/cache
+
+matrix:
+  include:
+    - php: 7.1
+      env: WP_VERSION=latest
+    - php: 7.0
+      env: WP_VERSION=latest
+    - php: 5.6
+      env: WP_VERSION=latest
+    - php: 5.6
+      env: WP_VERSION=trunk
+    - php: 5.6
+      env: WP_TRAVISCI=phpcs
+    - php: 5.3
+      env: WP_VERSION=latest
```

Comment:
See https://core.trac.wordpress.org/ticket/40407 - I don't think we need such a large matrix here.  We should include:

- PHP 5.2 (minimum supported WP version)
- PHP 5.6 (last 5.x version)
- PHP 7.1 (latest version)